README.md

```diff
@@ -364,6 +380,17 @@ Since such messaging doesn't require any server-side adaptation, it's better
 for this case to use the `getUserAgent` method in order to retrieve the
   required information.
 
+## Download of appropriate binary executables
+Some sites are used to download binary executables of native applications, and
+need to be able to propose the right binary to the user by default.  The right
+binary executable for the current user depends on a few factors: their
+operating system, its version, as well as their CPU architecture.
+
+In order to tackle that use case, download sites can opt-in to receive the
+`UA-Platform`, `UA-Platform-Version` and `UA-Architecture` hints (or query them
+through the API), in order to ensure the right binary is offered to the user by
+default.
```

Comment:
Non-seriously: request UA-Platform-LinuxPackageFormat

Reply:
There's certainly a use-case there, but since it's not something the current UA supports, I think it's something we can think about further down the line, if there's demand